rules/retrieve.smk

```diff
@@ -728,6 +728,77 @@ if config["enable"]["retrieve"]:
 
+    rule seawater_temperature:
+        output:
+            seawater_temperature="data/seawater_temperature.nc",
+        log:
+            "logs/retrieve_seawater_data.log",
+        resources:
+            mem_mb=10000,
+        retries: 2
+        shell:
+            """
+            wget -nv -c https://zenodo.org/records/15198744/files/seawater_temperature.nc -O {output.seawater_temperature}
+            """
+
+    # dynamic inputs/outputs for hera data retrieval
+    hera_data_key = "be_2013-03-01_to_2013-03-08"
+    hera_data_url = storage(
+        f"https://zenodo.org/records/15828866/files/hera_{hera_data_key}.zip"
+    )
+
+    rule retrieve_hera_data_test_cutout:
+        input:
+            hera_data_url=storage(
+                f"https://zenodo.org/records/15828866/files/hera_be_2013-03-01_to_2013-03-08.zip"
+            ),
+        output:
+            river_discharge=f"data/hera_be_2013-03-01_to_2013-03-08/river_discharge_be_2013-03-01_to_2013-03-08.nc",
+            ambient_temperature=f"data/hera_be_2013-03-01_to_2013-03-08/ambient_temp_be_2013-03-01_to_2013-03-08.nc",
+        params:
+            folder="data",
+        log:
+            "logs/retrieve_hera_data_test_cutout.log",
+        resources:
+            mem_mb=10000,
+        retries: 2
+        run:
+            unpack_archive(input[0], params.folder)
+
+    rule retrieve_hera_data_2013:
+        output:
+            river_discharge="data/hera_2013/river_discharge_2013.nc",
+            ambient_temperature="data/hera_2013/ambient_temp_2013.nc",
+        params:
+            snapshot_year="2013",
+        log:
+            "logs/retrieve_hera_data_2013.log",
+        resources:
+            mem_mb=10000,
+        retries: 2
+        shell:
+            """
+            wget -nv -c https://jeodpp.jrc.ec.europa.eu/ftp/jrc-opendata/CEMS-EFAS/HERA/VER1-0/Data/NetCDF/river_discharge/dis.HERA{params.snapshot_year}.nc -O {output.river_discharge}
+            wget -nv -c https://jeodpp.jrc.ec.europa.eu/ftp/jrc-opendata/CEMS-EFAS/HERA/VER1-0/Data/NetCDF/climate_inputs/ta6/ta6_{params.snapshot_year}.nc -O {output.ambient_temperature}
+            """
+
+    rule retrieve_hera_data_2019:
+        output:
+            river_discharge="data/hera_2019/river_discharge_2019.nc",
+            ambient_temperature="data/hera_2019/ambient_temp_2019.nc",
+        params:
+            snapshot_year="2019",
+        log:
+            "logs/retrieve_hera_data_2019.log",
+        resources:
+            mem_mb=10000,
+        retries: 2
+        shell:
+            """
+            wget -nv -c https://jeodpp.jrc.ec.europa.eu/ftp/jrc-opendata/CEMS-EFAS/HERA/VER1-0/Data/NetCDF/river_discharge/dis.HERA{params.snapshot_year}.nc -O {output.river_discharge}
+            wget -nv -c https://jeodpp.jrc.ec.europa.eu/ftp/jrc-opendata/CEMS-EFAS/HERA/VER1-0/Data/NetCDF/climate_inputs/ta6/ta6_{params.snapshot_year}.nc -O {output.ambient_temperature}
+            """
```

Comment:
The year should not be hard-coded here. Should be possible with wildcards.

Also, no `wget` allowed for Windows compatibility. Can use `storage(...)` option above or a Python-based method.